app/views/posts/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>Posts</h1>
+<%= render partial: "users/username", locals: { user: @posts[0].author} %>
+<ul class='user-posts'>
+<% @posts.each_with_index do |post, index| %>
+  <li>
+  <h4>Post #<%=index + 1%></h4>
+  <p><%=post.text%></p>
+  <span>Comments: <%= post.comments_counter%>, Likes: <%= post.likes_counter%></span>
+  </li>
```

Comment:
- [ ] Please make sure clicking on each post takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_single_post.png)